test_sets/policies_smoke.txt

```diff
@@ -0,0 +1,6 @@
+./test_scripts/Polices/build_options/001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_kilometers.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_days.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_IGN_cycles.lua
+./test_scripts/Polices/build_options/ATF_P_TC_PTU_Transfer_Several_Apps_Different_HMI_Levels_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_P_PTS_creation_rule_PROPRIETARY.lua
```

Comment:
@dboltovskyi, Additional general checks that can be done for Proprietary flow:
Correct policy assignment of application:
The scripts that I found and can be used for Proprietary flow(if data1.result.isSDLAllowed ~= true then) are:
ATF_P_TC_Register_App_Interface_Assign_Default_Policies_To_Application_Which_Appid_Does_Not_Exist_In_LPT
ATF_P_TC_Register_App_Interface_Assign_Existing_Policies_To_Application_Which_Appid_Exists_In_LPT